safenode/src/network/msg/mod.rs

```diff
@@ -54,7 +54,7 @@ impl SwarmDriver {
                 let _ = self
                     .pending_requests
                     .remove(&request_id)
-                    .ok_or(Error::Other("Request to still be pending.".to_string()))?
+                    .ok_or(Error::ReceivedResponseDropped(request_id))?
                     .send(Err(error.into()));
```

Comment:
Same here, we can probably map the error to `InternalMsgChannelDropped` instead of ignoring the result